.gitpod.yml

```
@@ -1,4 +1,4 @@
-image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-trigger.4
+image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-devtest.5
```

What do you think about simplifying our dev setup like so?

```suggestion
image:
  file: dev/image/Dockerfile
```

---

done